cmd/cortex/main.go

```diff
@@ -139,6 +143,12 @@ func main() {
 		prometheus.MustRegister(resourceWatcher)
 	}
 
+	tracer, err := loki.NewTracer(loki.DefaultConfig)
```

Comment:
In Prometheus, they are defined in the configuration file. Here, we're using the defaults in Loki with no way of customizing them.